client/redux/reducers/metadata.js

```diff
@@ -98,8 +98,9 @@ export default (state = initialState, action) => {
       };
     case types.GET_METADATA_FAILURE: {
       const {
-        response: { status },
-        message,
+        // response: { status },
+        status,
```

Comment:
Are we extracting `status` and also assigning it to `message` here?

Reply:
[Image of console error](https://imgur.com/a/tI0bg0m)

Thanks for checking. It seems the problem is that event.response returns undefined sometimes. Therefore the status property does not exist for an undefined object. I had to update this code to the following (the one you reviewed is not ideal). The updated code will look like this:

from:
`client/redux/reducers/sagas/metadata.js`

```
  } catch (e) {
    yield put(getMetadataFailure(e));
  }

```

input variable e, the event, gets called into getMetadataFailure and the data flows into the reducer (below), where it destructures into `response` and `message` variables. in the return statement, `code` value gets set to response.status if response exists (if status is undefined, it will return undefined). if response doesn't exist, it will return response, which should be undefined. I'm sure there's a cleaner way to do this so, please let me know if you have any suggestions. But for now, I am committing these two changes in the next push.

reducer:
`client/redux/reducers/metadata.js`
```
    case types.GET_METADATA_FAILURE: {
      const {
        response,
        message
      } = action.payload;

      return {
        ...state,
        error: {
          code: !!response ? response.status : response,
          message,
          error: action.payload,
        },
      };
    }
```

Reply:
Thanks for this explanation. Do you know why `event.response` is undefined sometimes? That would be something worth fixing, if possible.

It seems like this issue might be more complicated than we thought, so I prefer not to lump the fix in with this PR and open a separate issue and PR. This will make each PR easier to test and review.

Reply:
I'm not certain if it is the only time it happens but `event.response` certainly returns` undefined` when you start the client without internet connection. I did so to test whether the error toast would display correctly (which it does) but received this console error. 

Just to be clear, for now you want me to revert back all changes to the metadata.js files to the original code in the `dev` branch in my next push?

Reply:
as discussed, reverted metadata.js files back to dev branch versions in latest push